packages/@aws-cdk/aws-scheduler-targets-alpha/lib/target.ts

```diff
@@ -68,7 +68,7 @@ export interface ScheduleTargetBaseProps {
 export abstract class ScheduleTargetBase {
 
   constructor(
-    private readonly baseProps: ScheduleTargetBaseProps,
+    protected readonly baseProps: ScheduleTargetBaseProps,
```

Comment:
after some of these changes i suspect we can revert this line and the rest of the changes in this file

Reply:
Reverted the visibility changes back with the introduction of the method `overrideTargetConfig`